libraries/mapbox_install.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Performs install/uninstall methods for the Mapbox plugin
+ *
+ * PHP version 5
+ * LICENSE: This source file is subject to LGPL license
+ * that is available through the world-wide-web at the following URI:
+ * http://www.gnu.org/copyleft/lesser.html
+ * @author	   Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi - http://source.ushahididev.com
+ * @module	   Actionable Installer
+ * @copyright  Ushahidi - http://www.ushahidi.com
+ * @license    http://www.gnu.org/copyleft/lesser.html GNU Lesser General Public License (LGPL)
+ */
+
+class Mapbox_Install {
+
+	/**
+	 * Constructor to load the shared database library
+	 */
+	public function __construct()
+	{
+		$this->db = Database::instance();
+	}
+
+	/**
+	 * Creates the required database tables for the Mapbox plugin
+	 */
+	public function run_install()
+	{
+		// Create the database tables.
+		// Also include table_prefix in name
+		$this->db->query('CREATE TABLE `'.Kohana::config('database.default.table_prefix').'mapbox` (
```

Comment:
This could all just go into the config table.. The point of having a generic key/value config table was to avoid all these plugin specific tables :)